test/Twig/Tests/Profiler/Dumper/AbstractTest.php

```diff
@@ -32,13 +32,14 @@ protected function getProfile()
 
         $embedded = clone $embedded;
         $index->addProfile($embedded);
-        $a = range(1, 1000);
+        usleep(500);
         $embedded->leave();
-        $profile->leave();
```

Comment:
Im not sure why to leave the `profile` before leaving the `index`, from a DX point of view I would say this a state exception, but maybe I'm not to good in reading this :)